.golangci-maximal.yml

```diff
@@ -0,0 +1,182 @@
+linters:
+  disable-all: true
+  enable:
+    - copyloopvar
+    - cyclop
+    - depguard
+    - dupword
+    - errcheck
+    - errname
+    - errorlint
+    - exhaustruct
+    - exptostd
+    - funlen
```

Comment:
The default function length is 60 chars (including func params), which is extremely low. It complains about almost every test function by default. I wouldn't recommend this linter unless configured different than the default.

Reply:
You mean 60 _lines_, don't you? It doesn't check the length of the function name, AFAICT.

I think 60 lines is too much, not too little!

Reply:
You're right, I misunderstood what the the `funlen` linter checked. Nevertheless, many tests are table-driven and benefit from having the test cases located next to the test logic. In the Go Driver, the linter complains about most tests.